lib/constants/tiers-types.js

```diff
@@ -12,4 +12,12 @@ export const AmountTypes = {
   FLEXIBLE: 'FLEXIBLE',
 };
 
+export const CurrencyPrecision = {
+  DEFAULT: 2,
+};
+
+export const getPrecision = amount => {
+  return amount.toString().slice(-2) === '00' ? 0 : CurrencyPrecision.DEFAULT;
+};
```

Comment:
This file is in `constants` so it should be only about defining constants. This helper could be helpful for other parts of the code, what about putting it in `lib/utils.js` with a more explicit name like `getPrecisionFromAmount`?

The `CurrencyPrecision` doesn't seems to be in the right place neither, because amount precision is not only for tiers (orders, transactions, expenses have amounts too). I'm not convinced that this enum is required, but if you want to add it then it should live in `lib/constants/currency-precision.js`.

Reply:
I agree that they better belong in `lib/utils.js` & `lib/constants/currency-precision.js` .

As for the `CurrencyPrecision`, I wasn't sure of how to express it earlier and my initial assumption of currency precision only being up to 2 decimal places. But I thought that may change in the future with the possibility of maybe displaying bitcoin values and other digital currencies and how that may look like.

Reply:
> But I thought that may change in the future with the possibility of maybe displaying bitcoin values and other digital currencies and how that may look like.

Ah yes that's a very good argument. I can see how having a `CurrencyPrecision.BITCOIN_DEFAULT` could help in the future.